RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 20.1
 -----
 * [*] Site Creation: Enhances the design selection screen with recommended designs. [#18740]
+* [*] Block Editor: Bump react-native-gesture-handler to version 2.3.2. [#18742]
```

Comment:
I think we should use `[internal]` since it's not a user-facing change

```suggestion
* [*] [internal] Block Editor: Bump react-native-gesture-handler to version 2.3.2. [#18742]
```